config.go

```diff
@@ -1416,9 +1416,15 @@ func ValidateConfig(cfg Config, interceptor signal.Interceptor, fileParser,
 		return nil, mkErr("error validating logging config: %w", err)
 	}
 
-	cfg.SubLogMgr = build.NewSubLoggerManager(build.NewDefaultLogHandlers(
-		cfg.LogConfig, cfg.LogRotator,
-	)...)
+	// If a sub-log manager was not already created, then we'll create one
+	// now using the default log handlers.
+	if cfg.SubLogMgr == nil {
```

Comment:
Makes sense... The whole reason the `SubLogMgr` (or previously the log writer) is on the config struct in the first place is its usage in `litd`. Should've remembered that when reviewing :man_shrugging: